language/de.json

```diff
@@ -0,0 +1,134 @@
+{
+  "semantics": [
+    {
+      "label": "Medien",
+      "fields": [
+        {
+          "label": "Typ",
+          "options": [
+          ],
+          "description": "Optionales Medium, das über der Frage platziert werden kann"
+        }
+      ]
+    },
+    {
+      "label": "Aufgabenbeschreibung",
+      "default": "Einfügen der fehlenden Worte",
+      "description": "Eine Anleitung, die dem Benutzer erklärt, wie diese Aufgabe zu beantworten ist.",
+      "tags": [
```

Comment:
It is not necessary to include empty arrays in language files, values will be inherited from semantics.json